cocos/network/HttpRequest.h

```diff
@@ -342,6 +336,18 @@ class CC_DLL HttpRequest : public Ref
    		return _headers;
    	}
     
+private:
+    inline void doSetResponseCallback(Ref* pTarget, SEL_HttpResponse pSelector)
```

Comment:
why use another private function here... it's a bit wired...
What do you think?